numpy/ma/tests/test_core.py

```diff
@@ -1990,6 +1991,177 @@ def test_datafriendly_mul_arrays(self):
         assert_equal(a, [[1, 1], [3, 3]])
         assert_equal(a.mask, [[0, 1], [0, 1]])
 
+    def test_inplace_addition_scalar_uint8(self):
```

Comment:
The types could be done. As there were some tests for these operators already, which I borrowed for my case, maybe it is worth consolidating my tests with those if we want to loop through the types.

The operators would prove tricky. It's possible we could zip the inplace and non-inplace operators together and loop through them. However, true division had to be handled differently. Maybe you have some thoughts on how I could improve that case. If so, please let me know.